drivers/metal/rendering_device_driver_metal.mm

```diff
@@ -2293,6 +2302,8 @@ void deserialize(BufReader &p_reader) {
 
 		ShaderStageData stage_data;
 		stage_data.stage = v.shader_stage;
+		stage_data.is_position_invariant = compiler.is_position_invariant();
+		stage_data.supports_fast_math = !entry_point.flags.get(spv::ExecutionModeSignedZeroInfNanPreserve);
```

Comment:
I assume this is unrelated, but safe?

Reply:
It is, and incidentally, MoltenVK defaults to always enabling fast math, so I've done the same.